MoreLinq.Test/CreateTest.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace MoreLinq.Test
+{
+    [TestFixture]
+    public class CreateTest
+    {
+        [Test]
+        public void CreateIsLazy()
+        {
+            Assert.That(MoreEnumerable.Create(BreakingFunc.Of<IEnumerator<object>>()), Is.Not.Null);
+        }
+
+        [Test]
+        public void CreateYieldsEnumeratorElements()
+        {
+            var xs = Enumerable.Range(1, 10);
+            var result = MoreEnumerable.Create(() => xs.GetEnumerator());
+            Assert.That(result, Is.EquivalentTo(xs));
+        }
+
+        [Test]
+        public void CreateDisposesEnumerator()
```

Comment:
There should be a test that it is disposed even if the sequence is not read entirely (as long as the wrapper is disposed).